lib/main.dart

```diff
@@ -234,6 +234,22 @@ class _ThunderAppState extends State<ThunderApp> {
               theme = theme.copyWith(pageTransitionsTheme: pageTransitionsTheme);
```

Comment:
I think perhaps moving the additional styling into here would make more sense (since we're already modifying the themes here)!

Reply:
Done!